pywemo/ssdp.py

```diff
@@ -215,40 +219,47 @@ def scan(st=None, timeout=DISCOVER_TIMEOUT, max_entries=None, match_mac=None):
     start = calc_now()
 
     try:
+        """
         sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-
+        sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, 2)
+        sock.bind(('192.168.4.134', 8012))
```

Comment:
Is that helpful in some way? It's just a bit of commented out code that won't run on anyone's system. I would just remove it. Commented-out code is evil :)